src/database/migrations/1739918380436_member_user_public_key.ts

```diff
@@ -0,0 +1,19 @@
+import { type Kysely, sql } from 'kysely'
```

Comment:
I used `_` in the filename here instead of `-` as well as the singular noun, but I wasn't sure what the convention is in the code base as there seems to already be a bit of inconsistency with `uuid_fn` and `study-runs` . Happy to change as needed :man_shrugging: .

Reply:
Yes, I used the CLI, but it was something like:

```bash
$ npx kysely migrate:make member_user_public_key
```

So, the "convention" is user provided vs kysely itself :man_shrugging: .